base/ltoutenc.dtx

```diff
@@ -3129,8 +3149,29 @@
 \DeclareTextSymbol{\textbaht}            \UnicodeEncodingName{"0E3F}
 \DeclareTextSymbol{\SS}                  \UnicodeEncodingName{"1E9E}
 \DeclareTextSymbol{\textcompwordmark}    \UnicodeEncodingName{"200C}
+%    \end{macrocode}
+%    
+% \changes{v2.0u}{2021/02/19}{Add \cs{textnonbreakinghyphen}, \cs{textfiguredash}
+%      and \cs{texthorizontalbar} (gh/404)}
+%    \begin{macrocode}
+\DeclareTextSymbol{\textnonbreakinghyphen} \UnicodeEncodingName{"2011}
+%\DeclareTextSymbol{\textfiguredash}      \UnicodeEncodingName{"2012}
 \DeclareTextSymbol{\textendash}          \UnicodeEncodingName{"2013}
 \DeclareTextSymbol{\textemdash}          \UnicodeEncodingName{"2014}
+%\DeclareTextSymbol{\texthorizontalbar}   \UnicodeEncodingName{"2015}
+%    \end{macrocode}
+%    Unfortunately many fonts do not implement \texttt{"2012} and/or
+%    \texttt{"2015} (including the \LaTeX{} default fonts for Unicode
+%    engines) so we provide some approximations if the glyph is
+%    missing, like we do for \texttt{OT1} and \texttt{T1}.
+%    \begin{macrocode}
+\DeclareTextCommand{\textfiguredash}     \UnicodeEncodingName
+     {\iffontchar\font "2012 \char "2012 \else \char "2013 \fi}
+\DeclareTextCommand{\texthorizontalbar}     \UnicodeEncodingName
+     {\iffontchar\font "2015 \char "2015 \else \char "2014 \char "2014 \fi}
```

Comment:
I think we should only fallback to a single U+2014. That's consistent with the Unicode codecharts where both look identical, it looks better when used as quotation marker and it avoids gaps when fonts have sidebearings for U+2014.

Reply:
the more I look around the more I think fallback to a single em-dash is indeed the right thing